nixos/modules/services/desktop-managers/plasma6.nix

```diff
@@ -339,8 +341,32 @@ in
         capabilities = "cap_sys_nice+ep";
         source = "${lib.getBin pkgs.kdePackages.kwin}/bin/kwin_wayland";
       };
+
+      ksystemstats_intel_helper = {
+        owner = "root";
+        group = "root";
+        capabilities = "cap_perfmon+ep";
+        source = "${pkgs.kdePackages.ksystemstats}/libexec/ksystemstats_intel_helper";
+      };
+
+      ksgrd_network_helper = {
+        owner = "root";
+        group = "root";
+        capabilities = "cap_net_raw+ep";
+        source = "${pkgs.kdePackages.libksysguard}/libexec/ksysguard/ksgrd_network_helper";
+      };
     };
 
+    # Recommended by upstream
```

Comment:
but that doesn't really work on nixos anyway 😅 

```
 ➜ sudo timedatectl set-ntp true
Failed to set ntp: Changing system settings via systemd is not supported on NixOS.
```

```
 ➜ sudo timedatectl set-timezone true
Failed to set time zone: Changing timezone via systemd is not supported when it is set in NixOS configuration.
```

And I think the sample generated config already has a timezone of Europe/Amsterdam hinting people really hard to correct it.